src/main/java/org/bookstore/dto/BookDto.java

```diff
@@ -0,0 +1,15 @@
+package org.bookstore.dto;
+
+import java.math.BigDecimal;
+import lombok.Data;
+
+@Data
+public class BookDto {
```

Comment:
it is better to use records for DTO

if your Checkstyle fairs at them - upgrade `maven-checkstyle-plugin` to 3.3.0